tests/functests/instancetype_test.go

```diff
@@ -80,7 +86,7 @@ var _ = Describe("Common instance types func tests", func() {
 
 				var completePreferences []instancetypev1beta1.VirtualMachineClusterPreference
 				for _, preference := range clusterPreferences.Items {
-					if preference.Spec.Requirements != nil {
```

Comment:
We should keep this check, can you make it an `Expect(preference.Spec.Requirements).ToNot(BeNil())` if it is not skipped.

Reply:
You right, nice idea. Done.